### DISCUSSION TITLE
Exchanges wont send Iota

### DISCUSSION THREAD

**Discussion by Taberite** [iotaledger/firefly#2550] [Help & Support] [answered]
I am trying to send Miota from Defi-Crypto.com.
I am able to generate a wallet address on Firefly but none of the exchanges are recognizing the address which starts with iota.......  The error states that I am trying to send the wrong type of coin to firefly.  I am doing this as a BEP20.  Please tell me what I am doing wrong.  Thank you!

**Comment by MiKeF05** [comment, marked as answer]
You can't send wrapped BEP20 tokens to the IOTA network. You need to buy real IOTA tokens and not wrapped tokens.

**Reply by Taberite** [reply]
Mike,  thank you for the reply.  Your reply is over my head.  is there a dex i can "unwrap them at"?

**Reply by MiKeF05** [reply]
Unfortunatly there is no dex right now to buy real IOTA tokens. You only have the chance to unwrap them on an exchange.

**Reply by Taberite** [reply]
ok-  I bought them on Binance and sent them to Defi-crypto.com.  I can buy new ones from Binance and send to Firefly but it seems I can't send any of the ones I have on defi-crypto.com back to Binance.  Do I have any options?

**Reply by Taberite** [reply]
Also, I dont understand how i bought "wrapped" ones.  I thought Miota means that I own 1 Million Iota.  or maybe what you are saying is Binance put them on Bep20 which means they are 'wrapped".  I havent found anywhere else to purchase Iota.  Where do you purchase "unwrapped" iota?  Thanks a bunch!

**Reply by MiKeF05** [reply]
Normally you should be able to buy iota tokens on Binance. You just need to make sure to send them to the IOTA network and not to the Binance network. Like you said BEP20 are wrapped tokens which are on the Binance network.

**Reply by Taberite** [reply]
I think I figured it out.  I have to send them back to Binance and then out to Firefly.  Thanks again
